### ISSUE TITLE
Long words don't break lines

### ISSUE THREAD

**Issue by VitorLuizC** [Frondor/vue-line-clamp#20]
Long words don't break lines.

## Example
```vue
<template>
  <p class="text" v-line-clamp:16="7">asdadsadsadsadsadasdasdasdasdasdasdasadsadsadsadsadasdasdasdasadsadsadsadasdasdasdasdasadsadsadsadasdasdasadsadsadasdasadsadsadsadasdasdasadsadasdasadsadasdasdasadsadasdasadsadsadasdasdasadsadasdasdasadsadsadsasa</p>
</template>
```

Causes

<img width="383" alt="captura de tela 2018-12-04 as 15 59 25" src="https://user-images.githubusercontent.com/9027363/49463168-ee714080-f7de-11e8-89bf-12b3db7e9aa3.png">

**Comment by VitorLuizC** [comment]
@Frondor I've already fix it on #21. Can you check and merge It?

**Comment by Frondor** [comment]
I'm sorry Vitor, I've been way too busy lately. I promise I'll address your issue tonight!
Thanks a lot!

**Comment by VitorLuizC** [comment]
@Frondor, no problem. I just commented because I had already fixed it in PR #21 and even then you put the label of "good first issue"

**Comment by Frondor** [comment]
Yes, the label was just a reminder for myself.
I was about to change the build system because I got a new Mac and rollup was giving me way too many problems, so before merging your PR I implemented another [project](https://github.com/Frondor/javascript-library-boilerplate) of mine in the branch [`new-build`](https://github.com/Frondor/vue-line-clamp/tree/new-build). You can check it out, and I'm merging you there.

I'd appreciate if you confirm the build at `vue-line-clamp@next` is working for your project. As it does here https://codepen.io/anon/pen/BvjVWv
